vizro-core/src/vizro/models/_components/form/date_picker.py

```diff
@@ -70,6 +70,12 @@ def build(self):
             inputs=inputs,
         )
 
+        clientside_callback(
+            ClientsideFunction(namespace="clientside", function_name="update_date_picker_position"),
+            output=Output(self.id, "dropdownPosition"),
+            inputs=Input(self.id, "n_clicks"),
+        )
+
```

Comment:
Could you add a comment above of why we need this work-around? And when we can eventually remove it? Link to issue on the mantine library if it's a bug on their side